common/src/main/java/com/khorn/terraincontrol/customobjects/CustomObjectStructure.java

```diff
@@ -97,7 +101,34 @@ public void spawnForChunk(ChunkCoordinate chunkCoordinate)
         {
             for (CustomObjectCoordinate coordObject : objectsInChunk)
             {
-                coordObject.spawnWithChecks(world, height, random);
+                if(coordObject.spawnWithChecks(world, height, random))
+                {
+                    BO3 bo3 = ((BO3)coordObject.getObject());
+                    if(bo3 == null)
+                    {
+                        throw new NotImplementedException();
```

Comment:
Can't we just use UnsupportedOperationException, instead of some internal `sun.reflect` class?